local-proxy/src/localServer.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+// tslint:disable:no-console
+import express from 'express';
+import { setupProxy } from './index';
+import program from 'commander';
+const NAME = 'reshuffle-local-server';
+program.option('-p, --port', 'listening port (3000 by default)');
+program.description(`Run the ${NAME} outside of create-react-app server`);
+
+function main() {
+  program.parse(process.argv);
+  // create the express app to route local requests
+  const app = express();
+
+  // reshuffle currently assumes running from src and uses
+  // a relative path resolution to reach backend
+  setupProxy(process.cwd() + '/src')(app);
```

Comment:
Does this work on windows? (just verifying)

Reply:
@vogre PING

Reply:
Yep! It does produce a strange path at disk root (`C://\src`, if you decide that your project root is on `C:\\`, but even that works).